src/catalog/src/builtin.rs

```diff
@@ -2281,7 +2281,7 @@ pub static MZ_COMMENTS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
     desc: RelationDesc::empty()
         .with_column("id", ScalarType::String.nullable(false))
         .with_column("object_type", ScalarType::String.nullable(false))
-        .with_column("object_sub_id", ScalarType::UInt64.nullable(true))
```

Comment:
We disallow indexes and MVs on catalog objects right? Otherwise we would have persisted files with the previous type.

Reply:
Exactly! When I try to create objects on top of the `mz_comments` table, this is what I get:
```
=> CREATE MATERIALIZED VIEW a AS (SELECT comment FROM mz_internal.mz_comments);
ERROR:  cannot create materialized view with unstable dependencies
DETAIL:  The object depends on the following unstable objects:
    mz_comments
```
or for an index...
```
=> CREATE DEFAULT INDEX ON mz_internal.mz_comments;
ERROR:  must be owner of TABLE mz_internal.mz_comments
```